.github/workflows/CI2.yml

```diff
@@ -108,14 +115,32 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
           cmake --build build --parallel 2
           cmake --install build
+      - name: Checkout arborx
+        if: ${{ matrix.arborx }}
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: arborx/ArborX
+          ref: master
+          path: arborx
+      - name: Build arborx
+        if: ${{ matrix.arborx }}
+        working-directory: arborx
```

Comment:
Want to fix my `cd build`?

Reply:
`cmake -B build` does that, is was a missing `\`